data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/petrifies/petrifies_ability.cfg

```diff
@@ -0,0 +1,108 @@
+#textdomain wesnoth-test
+
+#####
+# API(s) being tested: [petrifies]
+##
+# Actions:
+# Give the leaders a petrifies ability
+# Have side 1's leader attack side 2's leader with both their weapons
```

Comment:
Reminder to update the docs here

Reply:
What update is needed?

Reply:
Looking again, I think it works as-is. I thought that there was stuff missing, but rereading it after reading the other tests it seems to say enough. Maybe split it into
```
Have side 1's leader attack side 2's leader with their first weapon
Unpetrify side 2's leader
Have side 1's leader attack side 2's leader with their second weapon
```

Reply:
Updated.